.gitignore

```diff
@@ -20,6 +20,7 @@ Thumbs.db
 .tmproj
 *.sublime-project
 *.sublime-workspace
+*.lnk
```

Comment:
:) So whats the "squash" plan? 
I'll remove the line and I see in docs there is a .git/info/exclude file where I can use ignore the .gitignore changes. Should hopefully prevent this next time. I am using this to hide the shortcut to chrome with allowlocalfile access flag set  So yup totally unrelated.